docs/design/autoware-architecture/.pages

```diff
@@ -0,0 +1,10 @@
+nav:
+  - index.md
+  - Control
```

Comment:
Instead of putting empty files, I think it's good to show the page status as `under construction` like other pages.
https://github.com/autowarefoundation/autoware-documentation/blob/8c12ec64c0032a8c3bd60c3be2084a8fdf12c65f/docs/design/autoware-interfaces/index.md

![image](https://user-images.githubusercontent.com/31987104/164411020-3e1d2175-9ce4-471e-9d15-8479253838a9.png)

Reply:
Now your page looks like this, you can't even know the title.

https://autowarefoundation.github.io/autoware-documentation/pr-86/design/autoware-architecture/Control/

Reply:
fixed in 81d4bdcde01a5

Reply:
modify for warning
[4620564](https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/462056458ecde48490f4b3751cc37541597ff871)

Reply:
Could you use `Admonitions`? :pray: 
https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

Reply:
I did.
![image](https://user-images.githubusercontent.com/21360593/167342962-b0998d60-75c1-47ad-9ba9-89012d167319.png)

Reply:
Sorry, I didn't see the second commit. :bow: